### PR TITLE
Create webgpu_mrt_readback.html

### PR DESCRIPTION
**Pull request by Spiri0**
I have an example here for the readRenderTargetAsync function. The readRenderTargetAsync is of course not intended for such a purpose, but it is nice and clear.

Attention! This example only works with the PR #28197

### CODE REVIEW

**Comment by Mugen87** [comment]
Can you please rebase the PR so it refers to latest dev?

File: examples/webgpu_multiple_rendertargets_with_gpu_readback.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+				//Be careful with this value! 512 is already big. Reading data back from the GPU is computationally intensive
+				const size = 512;	
+				const renderTarget = new THREE.RenderTarget(size, size, { count: 2 });
+				renderer.setRenderTarget( renderTarget );
+				renderer.render( scene, camera );
+				renderer.setRenderTarget( null );
+
+				const width = renderTarget.width;
+          const height = renderTarget.height;
```
Comment (Mugen87) [inline examples/webgpu_multiple_rendertargets_with_gpu_readback.html:215]: Please make sure to fix such formatting issues.

File: examples/webgpu_multiple_rendertargets_with_gpu_readback.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+				
+				const pixelBuffer0 = await renderer.readRenderTargetPixelsAsync(renderTarget, 0, 0, width, height, 0);	//zero is optional
+				const pixelBuffer1 = await renderer.readRenderTargetPixelsAsync(renderTarget, 0, 0, width, height, 1);
+
+				const pixelBufferTexture0 = new THREE.DataTexture(pixelBuffer0, width, height);
```
Comment (Mugen87) [inline examples/webgpu_multiple_rendertargets_with_gpu_readback.html:220]: Please adhere the code style. Lines like this should look like so:
```js
const pixelBufferTexture0 = new THREE.DataTexture( pixelBuffer0, width, height );
```
Your editor should automatically fix such style issues when ESLint is proper supported.

**Comment by Spiri0** [comment]
I fixed the formatting and also renamed the example and entered the example in the json. The topic with the rebase and the screenshot is probably only possible with the console. To do this I need to have a local github dev repository. I have to deal with that first because I haven't had much to do with it yet.

**Comment by Spiri0** [comment]
I'm closing this PR because I redid the  whole PR process. It was a lot of effort to do this for the first time, but it was good for understanding

**Pull request closed without merging**